packages/package_info_plus/package_info_plus_platform_interface/lib/package_info_data.dart

```diff
@@ -9,6 +9,7 @@ class PackageInfoData {
     required this.version,
     required this.buildNumber,
     required this.buildSignature,
+    required this.installerStore,
```

Comment:
Why do we need to make it `required`, considering that we return this parameter only on Android, iOS and MacOS platforms? I would go with optional parameter instead.

Reply:
I have no special reason for making it required. I'll make it optional then.

Reply:
Awesome, thanks. With `required` it feels misleading for Linux, Web and Windows users to have this new parameter being added, but at the same time being not usable for them.